### PR TITLE
updated common component dependency

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5244

*Description of changes:*
- updated sbc-common-components to 2.4.2
- updated version to 1.4.0
- updated cached AddressComplete files
- fixed app title
- fixed exception when there is no user contact/email
- fixed errors when running locally with no LD key

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: public/config/configuration.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "SBC_CONFIG_AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
-  "ADDRESS_COMPLETE_KEY": "_secret_",
-  "LD_CLIENT_ID": "_secret_",
+  "ADDRESS_COMPLETE_KEY": "",
```
Comment (severinbeauvais) [inline public/config/configuration.json:10]: It was previously trying to load AddressComplete with a bum key.

File: public/index.html [hunk 2]
```diff
@@ -4,15 +4,18 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <title>Incorporation Application</title>
+    <title>BC Business Registry</title>
```
Comment (severinbeauvais) [inline public/index.html:7]: The user experience should be the same between our apps (they should not know they're switching apps).

File: src/main.ts [hunk 3]
```diff
@@ -50,7 +50,9 @@ async function start () {
   })
 
   // initialize Launch Darkly
-  await initLDClient()
+  if (window['ldClientId']) {
```
Comment (severinbeauvais) [inline src/main.ts:53]: This is to avoid console errors if there is no LD key.

Comment (severinbeauvais) [reply src/main.ts:53]: (Same code as Filings UI.)

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**